src/macos/cpid_macos.c

```diff
@@ -0,0 +1,288 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <ctype.h>
+
+#include <IOKit/IOKitLib.h>
+#include <sys/sysctl.h>
+#include <openssl/evp.h>
+
+#include "cpid/cpid_macos.h"
+#include "../constants.h"
+
+#define KERNEL_TASK_PID 0
+#define LAUNCHD_PID 1
+#define MACOS_SERIAL_NUMBER_BUFFER_SIZE 16
+#define MACOS_EXPECTED_DIGEST_INPUT_CONTENT_SIZE 88
+#define MAX_MICROS_OFFSET 999999
+#define OPEN_SSL_SUCCESS 1
+
+#pragma pack(push, 1)
+typedef struct {
+    uint64_t unix_epoch_seconds;
```

Comment:
Don't you need to include `<stdint.h>` to get the fixed width integer types? If it's working now, it's most likely just a happy side-effect of including another header.